src/cljs/vanilla/add_widget_modal.cljs

```diff
@@ -5,9 +5,9 @@
 
 
 (defn add-widget [new-widget selected-source]
-  ;(prn "add-widget " new-widget
-  ;  " //// selected-source " selected-source
-  ;  " //// keyword " (keyword (:keyword selected-source)))
+  (prn "add-widget " new-widget
```

Comment:
turning these back on can probably be undone, since things are working and we don't need to clutter the output

Reply:
Done